schemas/littlehorse/service.proto

```diff
@@ -80,6 +81,15 @@ service LittleHorse {
   // Runs a WfSpec to create a WfRun.
   rpc RunWf(RunWfRequest) returns (WfRun) {}
 
+  // Schedule repeated WfRun based on a cron expression
+  rpc ScheduleWf(ScheduleWfRequest) returns (ScheduledWfRunId) {}
+
+  // Search for existing schedules
+  rpc SearchScheduledWf(SearchScheduledWfRunsRequest) returns (ScheduledWfRunIdList) {}
```

Comment:
Propose: `rpc SearchScheduledWfRun`

And `SearchScheduledWfRunsRequest` --> `SearchScheduledWfRunRequest` (drops the `s`).